map/open-map-url/README.metadata.json

```
@@ -23,7 +23,8 @@
         "PortalItem"
     ],
     "snippets": [
-        "src/main/java/com/esri/samples/open_map_url/OpenMapURLSample.java"
+      "src/main/java/com/esri/samples/open_map_url/OpenMapURLSample.java",
+      "src/main/java/module-info.java"
```

```suggestion
      "src/main/java/com/esri/samples/open_map_url/OpenMapURLSample.java"
```
I don't think we've been putting module-info in here?

---

Removed. Will look at updating the script to exclude this from the metadata style checks.